### PR TITLE
doc: release-notes: Add security vulnerabilities

### PR DESCRIPTION
**Pull request by agross-oss**
This patch adds information about the security vulnerabilities being
fixed in the 1.14 release.

Signed-off-by: Andy Gross <andy.gross@linaro.org>

### CODE REVIEW

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-1.14.rst [hunk 1]
```diff
@@ -13,6 +13,22 @@ Major enhancements with this release include:
 
 The following sections provide detailed lists of changes by component.
 
+Security Vulnerability Related
```
[the remark this answers is not in the document]
Comment (agross-linaro) [reply doc/releases/release-notes-1.14.rst]: I'll change the wording to "have been addressed".  Any listing of vulnerabilities in the release notes indicate ones we have fixed.

As for the link to the CVE, it's a chicken and egg here.  We have the CVE IDs in hand, but they haven't been formally published.  Part of our process is that we don't publicly make these visible until they are part of a release.  The fixes are already in the code and have been for a while, but I was expecting to get the release notes out.  Then send in the CVE updates to mitre.

[the remark this answers is not in the document]
Comment (dbkinder) [reply doc/releases/release-notes-1.14.rst]: OK, let's leave off the link then (seemed like a good thing, but if they're "reserved" there's no further info there.

**Comment by dbkinder** [comment]
@agross-linaro Can we get the ``The following (CVE) security vulnerabilities have been addressed in this release:`` added before the bullet list so we can move this along?  I've got some more items to add here after this gets merged...

**Review by dbkinder — APPROVED** [review]
Hi @agross-linaro, I added a commit to your branch to address my comments, and add some x86 vulnerabilities addressed...

**Comment by dbkinder** [comment]
sigh, editing via the GitHub interface wasn't as clean as I was hoping...

File: doc/releases/release-notes-1.14.rst [hunk 2]
```diff
@@ -13,6 +13,39 @@ Major enhancements with this release include:
...
+
+* The shell DNS command can cause unpredictable results due to misuse of stack
+  variables.
+  CVE-2017-14201
+
+* The shell implementation does not protect against buffer overruns resulting
+  in unpredictable behavior.
+  CVE-2017-14202
+
+* We introduced Kernel Page Table Isolation, a technique for
```
Comment (andrewboie) [inline doc/releases/release-notes-1.14.rst]: This is the fix for Rogue Data Cache Load (CVE-2017-5754), can mention the CVE here.

**Comment by andrewboie** [comment]
thanks!

**Review by tyrantlizard77 — APPROVED** [review]

File: doc/releases/release-notes-1.14.rst (hunk 2 above)
Comment (dbkinder) [reply doc/releases/release-notes-1.14.rst]: I added the CVE and forced pushed the update.

**Review by nashif — APPROVED** [review]

**Pull request merged**